arbnode/node.go

```diff
@@ -552,6 +557,47 @@ func createNodeImpl(
 	if err != nil {
 		return nil, err
 	}
+
+	if config.EspressoCaffNode.Enable {
+		if exec, ok := exec.(*gethexec.ExecutionNode); ok {
+			espressoCaffNode := NewEspressoCaffNode(
+				func() *EspressoCaffNodeConfig { return &config.EspressoCaffNode },
+				exec.ExecEngine,
+				delayedBridge,
```

Comment:
It seems like this delayed bridge won't miss any messages because it starts from its genesis. Am I understanding that correctly?

Reply:
I think so. 
https://github.com/EspressoSystems/nitro-espresso-integration/blob/d976aff09c1ffd163b9f9fa3f43b43054bb3478e/cmd/nitro/init.go#L746-L751